src/server/utils/css.ts

```diff
@@ -76,4 +76,9 @@ export default `
     margin-bottom: 3px;
   }
   
+  table, th, td {
```

Comment:
funker det å legge på padding på td og th for å litt mer luft inn i kolonnene?
Eks:
th, td { padding: x-antall rem;}